### DISCUSSION TITLE
And suddenly invalid configuration with my DNS?

### DISCUSSION THREAD

**Discussion by nappalm** [vercel/community#7168] [Help] [answered]
### Summary

I had correctly configured my dns within the hostinger provider and vercel had accepted it perfectly.

Today I realized that my site was down, I entered vercel and in my dns configuration I had “invalid configuration”.

I don't understand how a couple of days ago everything was working fine, I have talked to hostinger support and they say it is a SSL problem on vercel's side. 

I have decided to change the dns for the record and the recommended CNAME. my current configuration inside hostinger is the following:
![image](https://github.com/vercel/community/assets/35016608/7c521093-9cdc-4c59-88de-9504db1cfaa6)

Now I don't know how my site stopped working, is it really a vercel problem?
![image](https://github.com/vercel/community/assets/35016608/dcb157a8-fa5b-418d-abdb-797d8fa26804)

---

It is worth mentioning that if I set the vercel dns inside hostinger I always get it:
![image](https://github.com/vercel/community/assets/35016608/b6d9a9dc-44a0-4be7-9ad4-f92e3b239dbe)

### Example

https://www.manuelmorales.dev

### Steps to Reproduce

My site: https://www.manuelmorales.dev

**Comment by amyegan** [comment]
Hey @nappalm. I'm seeing different values for these DNS records.

The [A record](https://www.whatsmydns.net/#A/manuelmorales.dev) for `manuelmorales.dev` has `15.197.130.221`. There is no [CNAME record](https://www.whatsmydns.net/#CNAME/www.manuelmorales.dev) set for `www.manuelmorales.dev`. And the [NS record](https://www.whatsmydns.net/#NS/manuelmorales.dev) shows 
```
ina1.registrar.eu
ina2.registrar.eu
ina3.registrar.eu
```

One quick thing to check is that there is no typo in the custom domain. The domain troubleshooting guide has some additional tips that should help you figure out what's gone wrong. 

https://vercel.com/docs/projects/domains/troubleshooting

**Reply by nappalm** [reply]
Thanks @amyegan,

Now I have set the nameservers directly, however vercel is still saying "configuration invalid" when previously it worked fine like this.
![image](https://github.com/vercel/community/assets/35016608/f6f8a55e-fd7f-49cd-ae7c-c2eed3d567eb)

I have an application that works in this way, it seems very strange to me that it has suddenly stopped working.

**Reply by amyegan** [reply, marked as answer]
When I [check the nameservers](https://www.whatsmydns.net/#NS/manuelmorales.dev), it's still shown as 
```
ina1.registrar.eu
ina2.registrar.eu
ina3.registrar.eu
```
with one server showing 
```
verify1.registrar.eu
verify2.registrar.eu
verify3.registrar.eu
```

This appears to be a problem with your registrar or DNS provider since your intended settings are not being reflected in the global DNS network. Someone in [a related discussion suggested](https://github.com/vercel/community/discussions/6921#discussioncomment-9752480) it could be a [serverHold](https://www.icann.org/resources/pages/epp-status-codes-2014-06-16-en#:~:text=lift%20the%20restriction.-,serverHold,-server%20hold). You should contact your registrar for help with this.

[a comment by an automated account is not shown]
